spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java

```diff
@@ -609,6 +612,24 @@ public void setSubBatchPerPartition(boolean subBatchPerPartition) {
 		this.subBatchPerPartition = subBatchPerPartition;
 	}
 
+	public Duration getAuthorizationExceptionRetryInterval() {
+		return this.authorizationExceptionRetryInterval;
+	}
+
+	/**
+	 * Set the interval between retries after {@code AuthorizationException} is thrown
+	 * by {@code KafkaConsumer}. By default the field is null and retries are disabled.
+	 * In such case the container will be stopped.
+	 *
+	 * The interval must be less than {@code max.poll.interval.ms} consumer property.
+	 *
+	 * @param authorizationExceptionRetryInterval the duration between retries
+	 * @since 2.4.0
```

Comment:
I will back-port this to the 2.3.x branch so I'll change these 2.4.0 references to 2.3.5.